interfaces/hotplug/spec.go

```diff
@@ -34,60 +34,52 @@ type Definer interface {
 
 // SlotSpec is a definition of the slot to create in response to udev event.
 type SlotSpec struct {
-	// XXX: Name is the name the interface wants to give to the slot; we
-	// might want to mediate this though (e.g. generate automatically), so this
-	// may change/go away.
+	// Name is the name the interface wants to give to the slot. It can be
+	// left empty, in such case hotplug machinery will auto-generate it.
+	// Hotplug machinery make sure the proposed name is unique by appending
+	// numeric suffix if needed.
 	Name  string
 	Label string
 	Attrs map[string]interface{}
 }
 
-// Specification contains data about all slots that a hotplug interface wants to have created in response to uevent.
+// Specification contains a slot definition to create in response to uevent.
```

Comment:
Maybe consider using `hotplug event` instead of uevent. It's probably to ok to keep `uevent` somewhere closer to where you process actual udev events.